contrib/gazebo/sawyer/pick_and_place.py

```diff
@@ -0,0 +1,33 @@
+"""
+Pick-and-place task for the sawyer robot
+"""
+
+from contrib.gazebo import gazebo_sawyer_env
+from rllab.core.serializable import Serializable
+
+
+class SawyerPickAndPlaceEnv(gazebo_sawyer_env.GazeboSawyerEnv, Serializable):
```

Comment:
With few exceptions, files should contain a single class, and the file name should match the class name.